libs/RobotKit/include/StateMachine.h

```diff
@@ -14,6 +14,8 @@ namespace sm::event {
 
 	struct setup_complete {
 	};
+	struct timeout {
```

Comment:
```suggestion
	struct sleep_timeout_did_end {
```

on trouve aussi `expire` plutôt que `end` dans la literature